packages/react/fast-components-react-base/src/data-grid/data-grid-row.schema.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Complies with FAST Tooling 1.0
```

Comment:
This looks the same as 2.0, you can just use one schema you dont need two unless you have to specify react children which is the only breaking change between 1 and 2.

Reply:
now with only one schema